core/finality-grandpa/src/communication/mod.rs

```diff
@@ -273,6 +296,11 @@ impl<B: BlockT, N: Network<B>> NetworkBridge<B, N> {
 		});
 
 		let topic = round_topic::<B>(round.0, set_id.0);
+
+        let mut votes_tally: HashMap<AuthorityId, VoteTally> = HashMap::new();
```

Comment:
spaces

Reply:
(not only here and not only in this file)